src/broker/content-cache.c

```diff
@@ -617,13 +617,13 @@ static void checkpoint_get_continuation (flux_future_t *f, void *arg)
         goto error;
 
     if (flux_respond (cache->h, msg, s) < 0)
-        flux_log_error (cache->h, "%s: flux_respond", __FUNCTION__);
+        flux_log_error (cache->h, "error routing checkpoint-get response");
```

Comment:
Why are some of these message "error routing" and others "error responding"?   There is not really any routing going on in the flux_respond call so IMHO the latter is more appropriate.